src/Components/Clock.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  time: string;
+};
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    time: (new Date()).toUTCString().slice(-12, -4),
+  };
+
+  timerIdChangeTime = 0;
+
+  componentDidMount() {
+    this.timerIdChangeTime = window.setInterval(() => {
+      this.setState({ time: (new Date()).toUTCString().slice(-12, -4) });
+      // eslint-disable-next-line no-console
+      console.info(this.state.time);
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Readonly<Props>) {
+    if (prevProps.name !== this.props.name) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${prevProps.name} to ${this.props.name}`);
+    }
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerIdChangeTime);
+  }
+
+  render() {
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
+          {this.props.name}
+        </strong>
+        <span>time is</span>
```

Comment:
great solution.
just add some styles and return spaces to clock)
```suggestion
        <span>{' time is '}</span>
```